core/src/main/scala/io/delta/tables/DeltaTableBuilder.scala

```diff
@@ -320,37 +322,30 @@ class DeltaTableBuilder private[tables](
       colNames.map(name => DeltaTableUtils.parseColToTransform(name))
     }.getOrElse(Seq.empty[Transform])
 
+    val tableSpec = org.apache.spark.sql.catalyst.plans.logical.TableSpec(
+      properties = this.properties,
```

Comment:
nit: remove `this.` (unless I'm mistaken, it's not used throughout the codebase)